apps/editor/src/js/ui/popupAddImage.js

```diff
@@ -108,26 +108,30 @@ class PopupAddImage extends LayerPopup {
     this.on('hidden', () => this._resetInputs());
 
     this.on(`change .${CLASS_IMAGE_FILE_INPUT}`, () => {
-      const filename = this._imageFileInput.value.split('\\').pop();
+      const filename = Array.prototype.map
```

Comment:
Why did you use `Array.prototype.map` instead of `this._imageFileInput.files.map()`?

Reply:
I understand. You did that because [`this._imageFileInput.files`](https://developer.mozilla.org/ko/docs/Web/API/FileList) is Array-like Object. If so, you can use the `toArray` function of the` tui-code-snippet` module rather than calling `Array.prototype.map` to clean it up.

```js
import toArray from 'tui-code-snippet/collection/toArray';
// ...

const filename = toArray(this._imageFileInput.files).map(/* */);
```

Reply:
`this._imageFileInput.files` is ArrayLike like `arguments`, we can't use `map` directly.

Reply:
> I understand. You did that because [`this._imageFileInput.files`](https://developer.mozilla.org/ko/docs/Web/API/FileList) is Array-like Object. If so, you can use the `toArray` function of the` tui-code-snippet` module rather than calling `Array.prototype.map` to clean it up.
> 
> ```js
> import toArray from 'tui-code-snippet/collection/toArray';
> 
> // ...
> const filename = toArray(this._imageFileInput.files).map(/* */);
> ```

okay, I will fix it.